pkg/storage/user.go

```diff
@@ -0,0 +1,109 @@
+package storage
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+
+	"go.mongodb.org/mongo-driver/bson"
+
+	"github.com/HarlamovBuldog/social-tournament-service/pkg/sts"
+)
+
+func (db *DB) AddUser(ctx context.Context, name string) (string, error) {
+	insertResult, err := db.conn.Collection(usersCollectionName).InsertOne(ctx, sts.User{
+		Name:    name,
+		Balance: 0,
+	})
+	if err != nil {
+		return "", fmt.Errorf("error insert doc to collection: %v", err)
```

Comment:
there is such a great package called https://godoc.org/github.com/pkg/errors I insist on using it's errors.Wrap instead fmt.Errorf

Reply:
not only here, but overall